cli/ops/fetch.rs

```diff
@@ -11,17 +11,24 @@ use futures::future::FutureExt;
 use http::header::HeaderName;
 use http::header::HeaderValue;
 use http::Method;
+use reqwest::Client;
 use std::convert::From;
+use std::path::PathBuf;
 
 pub fn init(i: &mut CoreIsolate, s: &State) {
   i.register_op("op_fetch", s.stateful_json_op2(op_fetch));
+  i.register_op(
+    "op_create_http_client",
+    s.stateful_json_op2(op_create_http_client),
+  );
 }
 
 #[derive(Deserialize)]
 struct FetchArgs {
```

Comment:
```suggestion
#[serde(rename_all = "camelCase")]
struct FetchArgs {
```
Then you're allowed to call it `clientRid` on the JS side.